composer.json

```diff
@@ -24,7 +24,7 @@
     "require": {
         "php": "^7.4|^8.0",
         "ext-mbstring": "*",
-        "imagine/imagine": "^1.2.4",
+        "imagine/imagine": "dev-develop",
```

Comment:
i would prefer to only use stable versions of upstream dependencies.

but happy to raise the version to the latest released major version.